src/components/LetterLoad.vue

```diff
@@ -62,11 +62,20 @@ export default {
   data () {
     return {
       date: new Date().toLocaleString(),
-      isSubmitted: true
+      isSubmitted: true,
+      additionalMessage: ""
     }
   },
   computed: {
-
+    persistCustomization : {
```

Comment:
Computed properties should only be used for reactive data without any side effects; if you look through some of the other files where computed properties are used, usually they will have little to no logic and will just return some value to make it reactive. I think persistCustomization should be a method. Perhaps something like this:
```
<template>
  <textarea v-model="customMessage"></textarea>
  <button @click="persistCustomization">Add customization</button>
</template>

export default {
  data() {
    return { customMessage: ' ' }
  },
  methods: {
    persistCustomization() {
      // Do something
    }
  }
}
```

Reply:
Thanks @DietBepis1 . I've made the fixes